docs/src/examples/components/List/Content/ListExampleContent.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 import { List } from '@stardust-ui/react'
 
-const items = [
-  'Program the sensor to the SAS alarm through the haptic SQL card!',
-  'Use the online FTP application to input the multi-byte application!',
-  'The GB pixel is down, navigate the virtual interface!',
-]
-
-const ListExample = () => <List items={items} />
+const ListExample = () => (
+  <List>
+    'Program the sensor to the SAS alarm through the haptic SQL card!', 'Use the online FTP
+    application to input the multi-byte application!', 'The GB pixel is down, navigate the virtual
+    interface!'
+  </List>
```

Comment:
`List` children should be `List.Item` components. Based on the name of this file, I think the intended markup is:

```jsx
<List>
  <List.Item content='...' />
  <List.Item content='...' />
  <List.Item content='...' />
</List>
```

Where `...` is the 3 comma separated strings.